.gitignore

```diff
@@ -1,3 +1,4 @@
+*.coverage*
```

Comment:
This ignores `.coveragerc` files and we want to keep those.

I think this might be causing your lint failure

Reply:
Can you change this @vkuzmin-uber?

Reply:
I will revert this line completely. We can update .gitignore in separate change if necessary.